src/lib/php/Data/License.php

```diff
@@ -33,13 +33,18 @@ class License extends LicenseRef
    * @var string
    */
   private $risk;
+  /**
+   * @var string
+   */
+  private $spdxCompatible;
 
-  function __construct($id, $shortName, $fullName, $risk, $text, $url)
+  function __construct($id, $shortName, $fullName, $risk, $text, $url, $spdxCompatible)
```

Comment:
The last argument could be made optional via
```
    function __construct($id, $shortName, $fullName, $risk, $text, $url, $spdxCompatible = false)
```

this fixes the breaking tests.